plugins/europeana/record.js

```diff
@@ -1,6 +1,27 @@
 import axios from 'axios';
 import omitBy from 'lodash/omitBy';
 
+const fieldLabels = {
+  dcContributor: 'Contributors',
+  dcCreator: 'Creators',
+  dcDescription: 'Description',
+  dcTitle: 'Title',
+  dcType: 'Type of object',
+  dctermsCreated: 'Creation date',
+  edmCountry: 'Providing country',
+  edmDataProvider: 'Providing institution',
+  edmRights: 'License of the media in this record (unless otherwise specified)'
+};
+
+/**
+ * Lookup the label for the provided metadata field key
+ * @key {String} key for metadata field
```

Comment:
`@param` not `@key`.

Strings are documented lowercase, i.e. `{string}` in JSDoc.